### ISSUE TITLE
Code actions for wrong capitalization

### ISSUE THREAD

**Issue by bettinaheim** [microsoft/qsharp-compiler#205]
I would be nice if code actions were provided in the case where a keyword or known identifier is misspelled e.g. lower case instead of upper case etc.

**Comment by bettinaheim** [comment]
Ping me if someone is interested in picking this up - I am happy to help.

**Comment by samarsha** [comment]
It would also be cool, in addition to the code action, if the error message showed a list of suggestions for unknown identifiers.

**Comment by cgranade** [comment]
@samarsha: Agreed entirely. Unknown identifier is by far the most common error in IQ#, for instance, but there's not much to help users out at that point, unfortunately.
